src/ioc/index.ts

```diff
@@ -1,12 +1,21 @@
 import IoCContainer from 'ioc-lite';
 
-// import { Logger } from '../services/logger';
-// import { HTTP } from '../services/http';
-// import { Users } from '../services/users';
+import { Logger } from '../services/logger';
+import { HTTP } from '../services/http';
+import { Users } from '../services/users';
 
-export const createIoCContainer = () =>  {
+export const createIoCContainer = () => {
   const ioc = new IoCContainer();
-  // you can register some resources right now below...
+
+  ioc.registerClass('http', HTTP);
+  ioc.registerClass('users', Users);
+  ioc.registerClass('logger', Logger);
+  ioc.register('apiConfig', {
+    path: '/api',
+    resources: {
+      users: '/users',
+    },
+  });
```

Comment:
You should avoid hard-coding the configuration and use the one that comes from the server.
Please register `apiConfig` inside the "app" function in the index.ts file.